plugins/uefi/fu-uefi-device.c

```diff
@@ -420,6 +420,7 @@ fu_uefi_device_new_from_entry (const gchar *entry_path)
 	id = g_strdup_printf ("UEFI-%s-dev%" G_GUINT64_FORMAT,
 			      self->fw_class, self->fmp_hardware_instance);
 	fu_device_set_id (FU_DEVICE (self), id);
+	fu_device_set_platform_id (FU_DEVICE (self), "/sys/firmware/efi/esrt/");
```

Comment:
Maybe just use `entry_path`? The platform ID should be unique for all devices too.

Reply:
Yep, adjusted